__tests__/app.test.js

```diff
@@ -26,28 +26,26 @@ describe("GET/api/topics", () => {
       .get("/api/tpics")
       .expect(404)
       .then(({ body }) => {
-        expect.objectContaining({});
+        expect(body).toEqual({});
       });
   });
 });
 
-// describe("GET/api/articles/:article_id", () => {
-//   test("200: return article by id", () => {
```

Comment:
Hi Nathaniel, this test was good as it checks if you are getting the correct article with your request.